schemas/telemetry/frecency-update/frecency-update.4.schema.parquetmr.txt

```diff
@@ -2,6 +2,7 @@ message frecency-update {
   required binary clientId (UTF8);
   required binary type (UTF8);
   required group payload {
+    required long Timestamp;
```

Comment:
This is still in the payload nested field

This should be at the top level, and after chatting with whd, because this is heka metadata we need to append the field name with "metadata" (i.e. metadata_Timestamp)

Reply:
Like this?